dev-support/hbase-personality.sh

```diff
@@ -552,8 +552,8 @@ function hadoopcheck_rebuild
 
   # All supported Hadoop versions that we want to test the compilation with
   # See the Hadoop section on prereqs in the HBase Reference Guide
-  if [[ "${PATCH_BRANCH}" = branch-1.* ]] && [[ "${PATCH_BRANCH#branch-1.}" -lt "4" ]]; then
-    yetus_info "Setting Hadoop 2 versions to test based on before-branch-1.4 rules."
+  if [[ "${PATCH_BRANCH}" = branch-1.3 ]]; then
```

Comment:
What about branch-1.4? No need of a check for it?

Reply:
Just expand the hided(unchanged) code below, you will see the condition branch for branch-1.4...